fix(kaltura-player): use latest @applicaster/quick-brick-player package

vyarovoy-sc opened the pull request:
## Description

This PR fixes a Kaltura player bug where it is necessary to update the `@applicaster/quick-brick-player` package version every time changes were added to this package

<details>
  <summary>Platforms Affected</summary>
  
- [x] Android
- [x] Android TV
- [x] Apple TV (TVOS)
- [x] Fire TV (FireOS)
- [x] iOS
- [x] LG TV (WebOS)
- [x] Samsung TV (Tizen)
- [x] Web
</details>

inckie commented:
Tested beta version, but need to update was actually by design, so we do not need to update all players at once if something changes it the library. I.e. player should hide that it uses that library for implementation.

inckie commented:
@vyarovoy-sc In general I recommend to actually revert this change for the reasons above, I specifically asked it to be that way. But I suppose while we are doing refactoring, it will save us some time.

vyarovoy-sc commented:
@inckie But I still need the latest changes! Can I just change the plugin version to the new one?

vyarovoy-sc commented:
@f-roland 
Yes but that's exactly what I did in PR, I changed dependency to peer dependency. Without this change, a specific version of the plugin uses in the dependencies, which is specified in the `package.json`. I changed the QB version in the root `package.json` file because the core plugin of the specified version does not contain the `quick-brick-player` in the dependencies

https://app.circleci.com/pipelines/github/applicaster/Zapp-Frameworks/6267/workflows/03638854-c8fb-4299-a725-50e79177e2d4/jobs/10726

inckie commented:
I mostly afraid of situation when changing QB version will break players, as it already happens with some plugins, creating endless regressions out of nowhere. So with explicit dependency it means that person who did the bump did verify it works, and we will not found out about sudden issues later from customers.
In theory, this package should contain 'default' player implementation logic, and just make sure it conforms external player API, but do not depend on some internal QB areas (as much as possible).
Given that we only have 2 players, if @f-roland thinks it will be ok, we can use peer for now.

vyarovoy-sc commented:
@inckie Found potential issues that may arise when using the static version instead of the latest. If you have updated the QB version in your application but the Kaltura Player version has not been updated for a long time, then there is a chance that you will get errors because `quick-brick-player` uses [peer dependencies with packages that match the QB version](https://github.com/applicaster/QuickBrick/blob/main/packages/quick-brick-player/package.json#L51).
I think we need to use latest version of the QB player in the dependency
